src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/SHAHashProvider.Browser.Managed.cs

```diff
@@ -154,94 +170,73 @@ private void InitializeState()
                 _stateSHA256[7] = 0x5be0cd19;
             }
 
+            public void Initialize()
+            {
+                this = new();
+            }
```

Comment:
SP-800-108 tests are failing on browser, and my suspicion falls on this. Doesn't the default constructor above zero all members?